app/main.py

```diff
@@ -1,2 +1,60 @@
+from dataclasses import dataclass
+from typing import Any, List, Optional, Hashable
+
+
+@dataclass
+class Node:
+    my_hash: int
+    key: int
+    value: Any
+
+
 class Dictionary:
-    pass
+    default_size = 8
+    load_factor = 2 / 3
+    re_size = 2
+
+    def __init__(self) -> None:
+        self.hashtable: List[Optional[Node]] = \
+            [None for _ in range(Dictionary.default_size)]
```

Comment:
it would be better to use:
```suggestion
              [None] * Dictionary.default_size
```